packages/faency/src/components/Switch.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { FC, ComponentProps } from 'react'
+import styled from 'styled-components'
+import css from '@styled-system/css'
+import omit from 'lodash.omit'
+import pick from 'lodash.pick'
+import { margin, MarginProps } from '@modulz/radix-system'
+import themeGet from '@styled-system/theme-get'
+
+export type SwitchProps = MarginProps & ComponentProps<'input'>
+
+const marginPropNames = margin.propNames
+
+export const Switch: FC<SwitchProps> = ({ children, ...props }) => {
+  const marginProps = pick(props, marginPropNames)
+  const inputProps = omit(props, marginPropNames)
+
+  return (
+    <SwitchWrapper {...marginProps} data-testid="switch">
+      <Input type="checkbox" {...inputProps} />
+      <FakeSwitch />
+    </SwitchWrapper>
+  )
+}
+
+const SwitchWrapper = styled('label')<MarginProps>(
+  css({
+    position: 'relative',
+    display: 'inline-block',
+    width: 6,
```

Comment:
I tried with `width: 5` (and used `space.2` in the Transform for the `:after` pseudo-component) and it seems to have a better look and feel, what do you think?

Width: 6  (Original)
![image](https://user-images.githubusercontent.com/38889364/76084996-e844b380-5f8f-11ea-84f8-e6769dd12f87.png)

Width: 5 (Suggestion)
![image](https://user-images.githubusercontent.com/38889364/76084964-d5ca7a00-5f8f-11ea-8fe0-b87e324468ee.png)

Reply:
Indeed I'm agree it looks better! :)